app/packages/spaces/src/types.ts

```diff
@@ -41,6 +41,7 @@ export type SpaceNodeJSON = {
   type?: SpaceNode["type"];
   pinned?: SpaceNode["pinned"];
   sizes?: number[];
+  _name?: string;
```

Comment:
wondering why `_name` instead of `name`?

Reply:
`name` is hidden in all situations, except when when it is a top-level "saved" workspace